src/main.py

```diff
@@ -20,19 +22,47 @@
 # Middleware to automatically open/close a database connection for every request
 @app.middleware('request')
 async def handle_request(request):
+    """Open database connection before each request is handled."""
     models.db.connect()
 
+
 @app.middleware('response')
 async def handle_response(request, response):
+    """Close database connection after each request is handled."""
     if not models.db.is_closed():
         models.db.close()
 
 
-
 @app.route('/store/', methods=['POST'])
 @doc.summary("store a new web id")
-async def r_store(req):\
-        # Get 'uri' and 'lblod_id' from JSON body and throw HTTP/400 if one of them is missing
+async def r_store(req):
+    """
+    Store a new webID in the database given a valid webID uri and a lblod uri.
+
+    Keyword arguments:
+    The request should contain valid json parameters for 'uri' and 'lblod'.
+        Example:
+            {
+                "uri": "https://jonasvervloet.inrupt.net/profile/card#me",
+                "lblod_id": "http://data.lblod.info/id/personen/41e449eafddf2c0c2365a294376780293d92fb401241589a1f403cdff8d2ce5a"
+            }
+
+    Returns:
+    The response contains json name/value pairs 'success', 'updated' and 'message'.
+        'success' denotes if the right query parameters are available and if they are valid.
+        'updated' denotes if the webID uri and lblod uri pair is stored in the database.
+            This is set to False when the webID uri is already used in the database.
+        'message' clarifies the response.
+
+        Example:
+            {
+                'success': True
```

Comment:
No comma's here and using both single & double quotes in the same thing is kind of ugly

Reply:
Which function is this?

Reply:
This is on the /store endpoint